### DISCUSSION TITLE
How to do form level validation ?

### DISCUSSION THREAD

**Discussion by p-clem** [ant-design/ant-design#32267] [Q&A]
I want to validate the entire collection of a form's fields together (e.g. to validate that the data makes sense together, not just that an individual field is valid).

Currently my only way is to have an input hidden field and add the validation rules there, but that's more like a hack. Is there another proper recommended way?

**Comment by ppodgorsek** [comment]
You might be looking for dynamic form rules: https://ant.design/components/form#form-demo-dynamic-rule
